bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -149,6 +149,12 @@ void ReplicaImp::onReportAboutInvalidMessage(MessageBase *msg, const char *reaso
 
 template <>
 void ReplicaImp::onMessage<ClientRequestMsg>(ClientRequestMsg *m) {
+  if (isSeqNumToStopAt(lastExecutedSeqNum)) {
+    LOG_INFO(GL, "Message ignored because we have to stop at super stable checkpoint");
```

Comment:
"Ignore ClientRequest message because ... " or similar.

Reply:
I would suggest, something more abstract even. Super stable checkpoint still seems to be a weird, opaque term that will likely need to be explained over and over again to the few people not dealing with it. I think something like the following is better.

"Ignoring ClientRequest because system is stopped at checkpoint pending control state operation (upgrade, etc...)"

Reply:
Adding this will also allow us to remove the comment below, which essentially becomes duplication. On a side note, please put comments above lines and not next to them. This style can cause problems with clang-format.